Add button to clone renku project

lorenzo-cavazzi opened the pull request:
Adds a button with the commands to clone the project locally, similar to what we find in both the GitLab and GitHub interfaces.

![Screenshot_20200928_205620](https://user-images.githubusercontent.com/43481553/94474035-54f1fa80-01cd-11eb-82c3-e2876be73262.png)

Preview: https://lorenzotest.dev.renku.ch/

fix #929

ciyer reviewed (changes requested):
Nice solution, but could we move it to the settings page, like this:

<img width="1229" alt="image" src="https://user-images.githubusercontent.com/1196411/94678168-5d0d7f80-031e-11eb-83f0-5e3ca6823997.png">


I think we want it out of the way because most (though not all) of our users would use the interactive  environments rather than cloning the project. We will want the screen real-estate where "clone" is for a "launch" button.

vfried commented:
A small comment regarding the change, if we put this in that tab it should look similar to the ssh and HTTP that are already there...

ciyer commented:
> A small comment regarding the change, if we put this in that tab it should look similar to the ssh and HTTP that are already there...

Definitely. I was just rushed for time with the mock-up. I will fix it.

lorenzo-cavazzi commented:
Sorry for the force push, I messed up a bit my local repo.
I removed the button and moved the clone info to the setting page. This is how it looks now

![Screenshot_20200930_143814](https://user-images.githubusercontent.com/43481553/94686355-02c6eb80-032b-11eb-8344-1e9bbea65898.png)

ciyer reviewed (approved):
Very nice!

ciyer reviewed (changes requested):
Wait spelling!

ciyer reviewed (approved):
Nice!
